### ISSUE TITLE
Getting started

### ISSUE THREAD

**Issue by angelaabbate** [FRBNY-DSGE/DSGE.jl#28]
Hi

I am new to Julia so I am afraid my question is very basic. I have downloaded Julia 0.7 and installed the DSGE package, with
    ` ] add DSGE` (as an alternative I used      ] add DSGE@0.7.2

This worked. Then I tried:
` using DSGE`

Then I had a LoadError: No deps.jl file could be found

What am I doing wrong? I would just try to run the Basic example file ;)

Thanks in advance

Angela

**Comment by chenwilliam77** [comment]
Hi Angela,

How did you install Julia? This  comment thread seems to generate a similar error with no deps.jl file found, and some of the comments indicate it may have to do with how you installted Julia. [See](https://github.com/JuliaStats/Distributions.jl/issues/755) .

Best,
William Chen

**Comment by rsarfati** [comment]
Hello! To follow up, try starting Julia with the flag --startup-file=no and then run `] build Arpack`. Let us know if that helped! If not, we can do further trouble-shooting.

**Comment by angelaabbate** [comment]
Hi to both of you

Thank you so much for your replies! I have not understood them though, sorry! ;) @chenwilliam77 I have downloaded Julia on Windows from the Julia Website (Version 0.7). Should I have done it through github? I checked the thread you mentioned but I did not see a solution (the user found it on his own).

@rsarfati How do I do that exactly? Should I start Julia from the command prompt? I have not managed to do that, I am not sure if it is a Problem with my work Computer, I keep investigating

Thank you in advance for your help!

Angela
